config.dev.toml

```diff
@@ -127,8 +127,13 @@ EnableCSRFProtection = false
     # Env override: ATHENS_NETRC_PATH
     NETRCPath = ""
 
-    # TraceExporterURL is the URL to which Athens populates distributed tracing 
-    # information such as Jaeger. 
+    # INSECURE_SOURCES takes in a comma separated list of sources that will
+    # apply a -insecure flag selectively when a go get is performed
+    # This is useful for private sources that do not support a secure protocol
+    InsecureSources = "gitlab.internal.com,bzr.internal.com"
```

Comment:
this should be a file, I believe that maintaining env might be uncomfortable.
with k8s it does not matter but for local, i think a file is readable. up for a discussion